tests/behat/features/api/mediapackage/mediafile_id_get.feature

```diff
@@ -0,0 +1,82 @@
+@api @media-package
+Feature: Get data from the media library in json format
+
+  Background:
+    Given there are media packages:
+      | id | name   | name_url |
+      | 1  | Looks  | looks    |
+      | 2  | Sounds | sounds   |
+    And there are media package categories:
+      | id | name    | package |
+      | 1  | Animals | Looks   |
+      | 2  | Fantasy | Sounds  |
+      | 3  | Space   | Looks   |
+
+    And there are media package files:
+      | id | name      | category | extension | active | file   | flavor     | author         |
+      | 1  | Dog 1     | Animals  | png       | 1      | 1.png  | pocketcode | Bob Schmidt    |
+      | 2  | Dog 2     | Fantasy  | mpga      | 1      | 2.mpga | pocketcode |                |
+      | 3  | Spaceship | Space    | png       | 0      | 3.png  |            | Micheal John   |
+      | 4  | Cat       | Animals  | png       | 1      | 4.png  | luna       |                |
+      | 5  | Ape       | Animals  | png       | 1      | 5.png  |            |                |
+      | 6  | Metroid   | Space    | png       | 1      | 6.png  | pocketcode | Jennifer Shawn |
+
+
+  Scenario: Requests with wrong HTTP_ACCEPT value should result in an error
+    Given I have a request header "HTTP_ACCEPT" with value "text/html"
+    And I request "GET" "/api/media/files"
+    Then the response status code should be "406"
+
+  Scenario: Getting file with id should return one media file with that id
+    Given I have a request header "HTTP_ACCEPT" with value "application/json"
+    And I request "GET" "/api/media/file/5"
```

Comment:
media/file/id

tests for 400 / 406 are missing

404 is missing in the CAPI specification.
@HCrane you might wanna add this.